### PR TITLE
Feat/homeview

### LINKED ISSUE #29: Add quote block to homeview and format welcome image with embedded logo

### PR DESCRIPTION
**Pull request by lkarpins**
** Resolves #29  **

** Describe the functionality to be tested: **
- Added media query to welcome image with logo
- Added quoteblock component below previewblock

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

**Review by Rallen13 — APPROVED** [review]
looks good

File: src/Components/QuoteBlock/QuoteBlock.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./QuoteBlock.scss";
+
+const QuoteBlock = () => {
+  return (
+    <>
+      <div className="quote-container">
+        <p>
+          "Abortion is health care. Abortion is freedom. Abortion is bodily
```
Comment (irmakerdem) [inline src/Components/QuoteBlock/QuoteBlock.js:9]: ❤️‍🩹

**Review by irmakerdem — APPROVED** [review]
Yay mission quote!

File: src/Views/HomeView/HomeView.scss [hunk 2]
```diff
@@ -36,3 +36,25 @@
...
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  padding: 2rem 0;
+  width: 100%;
+  gap: 1.5rem;
+}
+
+@media (min-width: 768px) {
```
Comment (catlynbowles) [inline src/Views/HomeView/HomeView.scss:51]: 💣.com

**Review by catlynbowles — APPROVED** [review]

**Pull request merged**